src/index.ts

```diff
@@ -28,7 +28,6 @@ export {
   SecretKey,
   EncryptedTreasureMap,
   HRAC,
-  TreasureMapBuilder,
```

Comment:
We may want to expose `TreasureMap` instead - `TreasureMapBuilder` was initially exposed on user request, who had a preference for manually building a `TreasureMap` instead of using `Alice.grant` prebuilt methods.